src/prototypes/email-signup/example/example-b.twig

```diff
@@ -0,0 +1,18 @@
+<div id="signup-form-b">
+  <div class="o-container--prose">
+    <div class="o-container__content">
+      <h2 class="heading">Get Weekly Digests</h2>
+      <label for="demo-input">Send weekly digests to...</label>
+      <demo class="demo-input-basic demo-input-basic--single-line">
+      <input
+        type="email"
+        value=""
+        class="c-input"
+        id="demo-input"
+        placeholder="Example: you@somewhere.tld"
+      />
+      <button class="c-button" type="button">Subscribe</button>
+      </demo>
```

Comment:
Did you mean to use a `<div>` instead of a `<demo>`? (Although the idea of a `demo` web component is kind of cool!)

Reply:
Lol that was just added in there when I copied the code but I'll change it